### PR TITLE
Updated Xcode Extension 1.2.0 | Added "Search Developer Documentation" command

### PR DESCRIPTION
**Pull request by SvenTiigi**
## Description

An update to the [Xcode Extension](https://www.raycast.com/SvenTiigi/xcode) to version 1.2.0:

- Small code cleanups and improvements
- Added README.md
- Added new "Search Developer Documentation" command
- Updated "Manage Simulators" command: Added accessory icons for Simulator states

## Type of change

- Bug fix / improvement for my extension

#### New "Search Developer Documentation" command

search-apple-developer-documentation

#### Added accessory icons to "Manage Simulators" command 

manage-simulators

## Checklist

- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I read the [extension guidelines](https://developers.raycast.com/information/review-process#guidelines)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

File: extensions/xcode/src/user-interfaces/apple-developer-documentation/apple-developer-documentation-list-item.user-interface.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+/**
+ * Apple Developer Documentation List Item
+ * @param appleDeveloperDocumentationEntry The AppleDeveloperDocumentationEntry
+ */
+export function appleDeveloperDocumentationListItem(
+  appleDeveloperDocumentationEntry: AppleDeveloperDocumentationEntry
+): JSX.Element {
+  return (
+    <List.Item
+      icon={icon(appleDeveloperDocumentationEntry)}
```
Comment (mattisssa) [inline extensions/xcode/src/user-interfaces/apple-developer-documentation/apple-developer-documentation-list-item.user-interface.tsx:13]:
```suggestion
      key={appleDeveloperDocumentationEntry.url}
      icon={icon(appleDeveloperDocumentationEntry)}
```

File: extensions/xcode/src/user-interfaces/xcode-simulators/xcode-simulators-list-item.user-interface.tsx [hunk 2]
```diff
@@ -19,12 +19,41 @@ export function xcodeSimulatorListItem(
       title={xcodeSimulator.name}
```
Comment (mattisssa) [inline extensions/xcode/src/user-interfaces/xcode-simulators/xcode-simulators-list-item.user-interface.tsx:19]: Missing `key` as well.

Comment (SvenTiigi) [reply extensions/xcode/src/user-interfaces/xcode-simulators/xcode-simulators-list-item.user-interface.tsx:19]: Seems like a `key` for the Xcode Simulator List Item has already been applied.

https://github.com/raycast/extensions/blob/8b826b4b8fcd0f1a8f107cf096342f50d1ba2943/extensions/xcode/src/user-interfaces/xcode-simulators/xcode-simulators-list-item.user-interface.tsx#L17

**Review by mattisssa — APPROVED** [review]

**Comment by SvenTiigi** [comment]
Thanks for the helpful comments and merge @mattisssa ✌️

**Pull request merged**